docs/deployment.md

```diff
@@ -34,7 +36,7 @@ Options:
   --port INTEGER                  Bind socket to this port.  [default: 8000]
   --uds TEXT                      Bind to a UNIX domain socket.
   --fd INTEGER                    Bind to socket from this file descriptor.
-  --reload                        Enable auto-reload.
+  --reload                        Enable auto-reload. (Disables --workers)
```

Comment:
I'm not sure we can have that here since it's automatically generated by click and our `tools.cli_usage` script may remove it
You need anyway to run the script and see, if the script removes it we wont be able and wont add it

Reply:
Ok, I will check the tool result... thanks for the hint

Reply:
@euri10 I'm having some trouble starting uvicorn from sources. Any idea how to easily resovle those dependency issues?
```
$ python uvicorn/main.py --help
Traceback (most recent call last):
  File "/workspaces/type-tree/uvicorn/uvicorn/main.py", line 1, in <module>
    import logging
  File "/workspaces/type-tree/uvicorn/uvicorn/logging.py", line 12, in <module>
    class ColourizedFormatter(logging.Formatter):
AttributeError: partially initialized module 'logging' has no attribute 'Formatter' (most likely due to a circular import)
```
I have installed all dependencies from requirements.txt. Any idea?
Thanks!

Reply:
you just need to run `./scripts/lint` and it will run the tools.cli_usage for you

that error is like wahoo, we should probably rename `logging.py` to avoid namespace conflicts, dont you think @Kludex ?

Reply:
I don't like the name of that module being the same as the stdlib, but I'm fine with it. The error above only happens because of bad usage.

Reply:
If you're strong about it, feel free to change it.

Reply:
yup never seen that error too, I was just wondering, I think a PR is worth it, I can't count the number of times where I myself got caught bad-naming modules this way, will do it

Reply:
script executed and changes pushed